ygnmi/ygnmi.go

```diff
@@ -258,13 +258,15 @@ type opt struct {
 	setFallback        bool
 	sampleInterval     uint64
 	datapointValidator ValidateFn
+	appendModuleName   bool
 	ft                 FunctionalTranslator
 }
 
 // resolveOpts applies all the options and returns a struct containing the result.
 func resolveOpts(opts []Option) *opt {
 	o := &opt{
-		encoding: gpb.Encoding_PROTO,
+		encoding:         gpb.Encoding_PROTO,
```

Comment:
Is there some way to actually set this to the other option that you added? I'm missing it if so :-)

Reply:
Ah, thanks.